.eslintrc.yml

```diff
@@ -71,6 +82,8 @@ rules:
   object-curly-spacing:
     - error
     - always
+    - objectsInObjects: false
+      arraysInObjects: false
```

Comment:
should read `- arraysInObjects: false` ?

Reply:
It should be like this because these exceptions are read as one attribute, even if there are two of them.

Reply:
Example: https://github.com/RedHatInsights/insights-advisor-frontend/blob/506bd384c4bc1902113de8b4445cf66620776b9f/.eslintrc.yml#L98